src/reports/Report.jsx

```diff
@@ -42,65 +40,37 @@ class Report extends React.Component {
   }
 
   render() {
-    if (this.state.error) {
-      const { params } = this.props.match
-
-      let alertHeading = 'Report not found'
-      if (params.stateId)
-        alertHeading = `Report ${params.reportId} for 2017 -> ${
-          params.stateId
-        } -> ${params.msaMdId} not found`
-      if (params.institutionId)
-        alertHeading = `Report ${params.reportId} for 2017 -> ${
-          params.institutionId
-        } -> ${params.msaMdId} not found`
-
-      return (
-        <div
-          className="usa-grid"
-          id="main-content"
-          style={{ marginTop: '3em' }}
-        >
-          <div className="usa-alert usa-alert-error" role="alert">
-            <div className="usa-alert-body">
-              <h3 className="usa-alert-heading">{alertHeading}</h3>
-              <p className="usa-alert-text">
-                Sorry, we couldn't find that report. Try to refresh the page. If
-                the problem persists please contact HMDA Help.
-              </p>
-            </div>
-          </div>
-        </div>
-      )
-    }
-
+    if (!this.state.isLoaded) return <LoadingIcon />
     if (this.state.report === null) return null
 
     const report = this.state.report
     const headingText = report
-      ? `Table ${report.table}: ${report.description}, ${report.year}`
+      ? `Table ${this.props.match.params.reportId.split('.txt')[0]}: ${
```

Comment:
I think we should just use the `report.table` value here, since that data is loaded.

Reply:
Ahh cool, updated.